conf/etc/v2/alert_filter_rules.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": "2.0.0",
+    "alert": {
+        "filter_type":"inclusion",
+        "resource_type":[
+            "enclosure:fru:controller",
+            "enclosure:fru:fan"
```

Comment:
Is there a separate task to expand this list? I think we need more inclusions.

Reply:
Actually, I had a discussion with Ajay P and decided to add only two resource types right now for testing and expand the list later.
@ajay-paratmandali  Do we have another task to expand the list?

Reply:
`Is there a separate task to expand this list? I think we need more inclusions.`
Yes investigation is separate task

Reply:
ok.